commands/staff/rm.js

```diff
@@ -15,7 +14,7 @@ module.exports = {
         args.forEach(async (x) => {
             let user = fn.getUser(x, message)
             if (user) {
-                let data = players.findOneAndUpdate({ user: user.id }, { $inc: { coins: amount } }).exec()
+                // let data = players.findOneAndUpdate({ user: user.id }, { $inc: { coins: amount } }).exec()
```

Comment:
Why are you commenting out the line that actually removes coins from the db

Reply:
data is not used in the following code

Reply:
It's used in error handling, and its chained to .exec() which saves in the database. Removing this line essentially makes the command 100% useless, since it doesn't do anything in the database anymore.

Reply:
ok

Reply:
done in [af2d2a1](https://github.com/wwosimulation/Narrator-bot/pull/85/commits/af2d2a15f6126e5001234f53d7f4be1e0ec06bbe)